cli/tests/e2e/snapshots/test_ci/test_github_ci_bad_base_sha/results.txt

```diff
@@ -132,6 +132,7 @@ Running semgrep-core with command:
 <MASKED>
 <MASKED>
 <MASKED>
+<MASKED>
```

Comment:
@kopecs did you add any lines that print to stderr? Probably innocuous but might be worth checking what this is exactly https://github.com/returntocorp/semgrep/blob/develop/cli/tests/e2e/test_ci.py#L534